src/session_pool/session.js

```diff
@@ -331,7 +331,7 @@ export class Session {
         return {
             name: toughCookie.key,
             value: toughCookie.value,
-            expires: new Date(toughCookie.expires).getTime(),
+            expires: new Date(toughCookie.expires).getTime() / 1000,
```

Comment:
So this means that `toughCookie` saves `expires` in millisecond precision but Puppeteer expects it in seconds? Would be worth a comment.

Reply:
Exactly. Both playwright and puppeteer expect seconds, while tough cookie have actually Date string here (to create which we have to use millis).